src/ConnectedField.js

```diff
@@ -44,19 +56,54 @@ const createConnectedField = ({ deepEqual, getIn }) => {
       return this.refs.renderedComponent
     }
 
+    handleChange(event) {
+      const { name, dispatch, parse, normalize, _reduxForm } = this.props
+      const value = onChangeValue(event, { name, parse, normalize })
+
+      dispatch(_reduxForm.change(name, value))
+    }
+
+    handleFocus() {
+      const { name, dispatch, _reduxForm } = this.props
+      dispatch(_reduxForm.focus(name))
+    }
+
+    handleBlur(event) {
+      const { name, dispatch, parse, normalize, _reduxForm } = this.props
+      const value = onChangeValue(event, { name, parse, normalize })
+
+      // dispatch blur action
+      dispatch(_reduxForm.blur(name, value))
+
+      // call post-blur callback
+      if (_reduxForm.asyncValidate) {
+        _reduxForm.asyncValidate(name, value)
+      }
+    }
+
+    handleDragStart(event) {
+      const { value } = this.props
+      event.dataTransfer.setData(dataKey, value == null ? '' : value)
+    }
+
+    handleDrop(event) {
+      const { name, dispatch, _reduxForm } = this.props
+      dispatch(_reduxForm.change(name, event.dataTransfer.getData(dataKey)))
+      event.preventDefault()
+    }
+
     render() {
-      const { component, withRef, name, _reduxForm, ...rest } = this.props
-      const { asyncValidate, blur, change, focus } = _reduxForm
+      const { component, withRef, name, ...rest } = this.props
       const { custom, ...props } = createFieldProps(getIn,
         name,
         {
           ...rest,
-          name,
```

Comment:
nope, must have been a refactoring missclick. I didn't want to touch any unnecessary things